cypress/support/e2e.ts

```diff
@@ -71,4 +71,5 @@ declare global {
 beforeEach(() => {
     cy.log("logging in, or getting session");
     login();
+    cy.visit("/");
```

Comment:
@shawn-hurley Please let me know if any objection in adding this here .

Reply:
no huge objections i was initially thinking that each test having that would be ok because it marks the actual start of the test, but it is also fine to do it here. I would just a note here or in the README/docs that every test starts by visiting `/` which should redirect to `/applications`